part2/stages/Functions/install-main

```diff
@@ -17,6 +17,9 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 #
 
+. ${DISK_CONF}
+# ^^^ defines TARGET_DISK
```

Comment:
Nit: please put your comments above the code you are explaining.

Reply:
Normally I would...but there are a number of files in installer.git that have this same "^^^ defines" structure, so I went with that for consistency.

Reply:
So it is used in 17 places where as the more traditional style is used more often. With that said, regardless if I disagree with that format, you are correct there is precedent for its usage.

Reply:
Well it's used specifically where configuration files are sourced. I'm not sure where that originated but I'm not a fan of it either. Just going with what I see.